console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.app.console.module.api.client.ui.dialog;
+
+import com.extjs.gxt.ui.client.event.Listener;
+import com.extjs.gxt.ui.client.event.MessageBoxEvent;
+import com.extjs.gxt.ui.client.widget.Dialog;
+import com.extjs.gxt.ui.client.widget.MessageBox;
+
+public class KapuaMessageBox extends MessageBox{
+
+    public static String question = "fa fa-question-circle fa-3x";
+    private Dialog dialog;
+    public static final String YESNO = Dialog.YESNO;
+
+    public static MessageBox confirm(String title, String msg, Listener<MessageBoxEvent> callback) { 
+          KapuaMessageBox box = new KapuaMessageBox();
```

Comment:
Have you tried removing completely this method, and then do both a `setIcon(question)` and a `setIconStyle(question)` inside `getDialog()`? I suspect you may achieve the same behaviour without this method.

Reply:
I have tried that but it doesn't work. :)